src/webview_view/queryResultPanel.ts

```diff
@@ -244,7 +246,9 @@ export class QueryResultPanel implements WebviewViewProvider {
     await commands.executeCommand(
       "workbench.view.extension.dbt_preview_results",
     );
-    this.transmitLoading();
+    if (this._panel) {
+      this.transmitLoading();
+    }
```

Comment:
to verify this error:
revert this change and start debugger. make sure to not select the "query results" tab at all. Go to an sql file and click on "execute sql". The query should run and query results tab should auto highlight. However, as in the debugger, you will see the error noted in issue #333. Adding this probably fixes the issue because the query results view has not been active yet in the session and transmitLoading is called before the _panel is set.